src/main/java/uk/gov/companieshouse/company_appointments/serialization/LocalDateSerializer.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.companieshouse.company_appointments.serialization;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import uk.gov.companieshouse.company_appointments.util.DateTimeFormatter;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+public class LocalDateSerializer extends JsonSerializer<LocalDate> {
+
+    @Override
+    public void serialize(LocalDate localDate, JsonGenerator jsonGenerator,
+                          SerializerProvider serializerProvider) throws IOException {
+        if (localDate == null) {
+            jsonGenerator.writeNull();
+        } else {
+            String format = DateTimeFormatter.format(localDate);
```

Comment:
The name `format` is a bit confusing. How about `formattedDate`?

Reply:
done